admin/services/pupil.service.js

```diff
@@ -87,6 +87,28 @@ const pupilService = {
       }
     }
     return true
+  },
+  /**
+   * Adds show Date of Birth flag for pupils with equal full names
+   * @param {Array} pupils
+   * @returns {Array}
+   */
+  requiresExtraFields: (pupils) => {
```

Comment:
This name seems a bit vague to me.  Perhaps something like `addIdentificationFlags` is more descriptive?